files/nrpe/check_object.py

```diff
@@ -277,12 +376,12 @@ def parse_command_line():
   parser.add_option("-p", "--password", dest='password', help='password')
   parser.add_option("-s", "--s3_access_key", dest='s3_access_key', help='username')
   parser.add_option("-k", "--s3_secret_key", dest='s3_secret_key', help='password')
-  parser.add_option("-l", "--s3_host", dest='s3_host', help='host')
+  parser.add_option("-b", "--s3_bucket_url", dest='s3_bucket_url', help='URL to S3 bucket')
+  parser.add_option("-l", "--s3_host", dest='s3_host', help='s3 host')
   parser.add_option("-t", "--tenant", dest='tenant', help='tenant name')
-
   parser.add_option("-d", "--debug", dest='debug', action='store_true', help='Debug mode. Enables logging')
-
-  parser.add_option("-b", "--s3_bucket_url", dest='s3_bucket_url', help='URL to S3 bucket')
```

Comment:
Please don't remove the `-b` parameter - it is used by another test (see check_object_s3_public.cfg ). Just add the new ones

Reply:
It is there. I did not remove it.

Reply:
Ah ok I see it now. Makes sense to pool the different kinds of args